website/config.yaml

```
@@ -1,6 +1,6 @@
 # Default page title
 title: "Hugo USWDS"
-baseURL: https://gsa.gov/hugo-uswds/ # The base URL of the site
+baseURL: https://localhost:1313/ # The base URL of the site
```

This looks like an unintended change, which may affect URL rewriting.
```suggestion
baseURL: gsa.gov/hugo-uswds/ # The base URL of the site
```

---

gsa.gov/hugo-uswds doesn't seem to resolve to a web page?
i was using the hugo serve to execute compliance tests against the local site, but i don't need to modify this if it causes an issue